### PR TITLE
Adding a conclusion

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: good-enough-practices-for-scientific-computing.tex [hunk 1]
```diff
@@ -1169,7 +1169,23 @@ \section{What We Left Out}\label{sec:omitted}
 
 \section{Conclusion}\label{sec:conclusion}
 
-\fixme{GW}{write a conclusion}
+We have outlined a series of practices for scientific computing based
+our collective experience, and the experience of the thousands of
+researchers we have met through Software Carpentry, Data Carpentry,
+and similar organizations.  These practices are pragmatic, accessible
+to people who consider themselves computing neophytes and can be
```
Comment (kcranston) [inline good-enough-practices-for-scientific-computing.tex]: (minor quibble): given the previous discussions of newcomers vs beginners vs novices, do we want to add yet another synonym (neophytes)?

File: good-enough-practices-for-scientific-computing.tex [hunk 2]
```diff
@@ -1169,7 +1169,23 @@ \section{What We Left Out}\label{sec:omitted}
...
+applied by both individuals and groups.
+
+Most importantly, we believe that practices like these are
```
Comment (kcranston) [inline good-enough-practices-for-scientific-computing.tex]: Should we first mention the benefits to the individual researcher (e.g. more efficient research) before the more altruistic reproducibility goals?

File: good-enough-practices-for-scientific-computing.tex [hunk 3]
```diff
@@ -873,10 +873,10 @@ \subsection*{What Not to Put Under Version Control}
 
 \section{Manuscripts}\label{sec:manuscripts}
 
-Gathering data, analyzing it, and figuring out what it means is the
-first 90\% of any project; writing up is the other 90\%. While writing
-is rarely addressed in discussions of scientific computing, computing
-has changed scientific writing just as much as it has changed research.
+An old joke says that doing the research is the first 90\% of any
+project; writing up is the other 90\%. While writing is rarely
+addressed in discussions of scientific computing, computing has
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:878]: Addresses #135 from @gcapes.

File: good-enough-practices-for-scientific-computing.tex [hunk 4]
```diff
@@ -1169,7 +1169,27 @@ \section{What We Left Out}\label{sec:omitted}
...
 
-\fixme{GW}{write a conclusion}
+We have outlined a series of practices for scientific computing based
+our collective experience, and the experience of the thousands of
+researchers we have met through Software Carpentry, Data Carpentry,
+and similar organizations.  These practices are pragmatic, accessible
+to people who consider themselves computing novices and can be applied
+by both individuals and groups.
+
+Most importantly, these practices make researchers more productive
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:1179]: Addresses comments by @kcranston

**Comment by kcranston** [comment]
Thanks, Greg!

**Pull request merged**